frontend/src/components/website/WebsiteForm.vue

```diff
@@ -0,0 +1,157 @@
+                        </div>
+                        <div class="inline-element">
+                            <label>Website Address</label>
+                            <VTextField
+                                name="address"
+                                placeholder="website address"
+                                single-line
+                                solo
+                                disabled
+                                readonly
+                                :value="currentWebsite.domain"
+                            />
+                        </div>
+                        <div class="inline-element">
+                            <label>SPA</label>
+                            <VSwitch
+                                :value="currentWebsite.single_page"
+                                readonly
+                                color="#3C57DE"
+                                inset
+                            />
+                        </div>
+                        <p class="inline-element">
+                            <span>Tracking ID: </span>
+                            <span>{{ currentWebsite.tracking_number }}</span>
+                        </p>
+                        <VBtn
+                            @click="update"
+                            color="#3C57DE"
+                            class="update-form-button"
+                        >
+                            Update
+                        </VBtn>
+                    </VForm>
+                    <div>
+                        <div>
+                            <h2>WebSite Tracking</h2>
+                            <p>
+                                This is Global Site tag tracking code for you website.
+                                Copy and Paste this code as the
+                                first item into the &lt;HEAD> of every Webpage you want to track
+                            </p>
+                            <TrackWebsite :tracking-number="currentWebsite.tracking_number" />
+                        </div>
+                    </div>
+                </VContainer>
+            </VFlex>
+        </VContent>
+    </VApp>
+</template>
+
+<script>
+    import TrackWebsite from './TrackWebsite.vue';
+    import { mapGetters, mapActions } from 'vuex';
+    import {GET_CURRENT_WEBSITE} from "../../store/modules/website/types/getters";
+    import {UPDATE_WEBSITE} from "../../store/modules/website/types/actions";
+
+    export default {
+        name: 'WebsiteForm',
+        components: {
+            TrackWebsite
+        },
+        data: () => ({
+            showErrorMessage: '',
+            showSuccessMessage: '',
+            nameRules: [
+                v => !!v || 'Website name is required',
+                v => (v && v.length >= 8) || 'Website name must be correct. Name must be at least 8 characters.'
+            ],
+            localWebsite: {
+                isWebsiteName: undefined
```

Comment:
Probably better to call it just websiteName, `is` prefix  usually used for bool fields